src/main/kotlin/slack/cli/buildkite/BuildkiteDataBindings.kt

```diff
@@ -0,0 +1,1460 @@
+@file:UseSerializers(JsonObjectAsMapSerializer::class, JsonElementKamlSerializer::class)
+
+package slack.cli.buildkite
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.UseSerializers
+import kotlinx.serialization.json.JsonArray
+import kotlinx.serialization.json.JsonObject
+
+/*
+ * Generated with https://app.quicktype.io/ using https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json.
+ *
+ * Modifications after:
+ * - Convert sealed classes to sealed interfaces
+ * - Convert sealed subtypes to value classes or data objects. This is important for the yaml to actually serialize correctly, otherwise there will be intermediate "value" levels.
+ * - Add @file:UseSerializers for JsonObjectAsMapSerializer and JsonElementKamlSerializer so that we can serialize the json types.
+ * - Change ExitStatusUnion.DoubleValue to a LongValue as exit codes aren't doubles.
+ * - Renames
+ *   - `CommandStep` -> `ScriptStep`
+ *   - `PurpleStepValue` -> `CommandStepValue`
```

Comment:
These color names are a bit funny, it's just how quicktype disambiguates repeated names. Most of these could be consolidated to a shared ref (but just aren't defined as such in the schema), so I manually fixed them here.